MiniSpace.Services.Posts/src/MiniSpace.Services.Posts.Infrastructure/Logging/MessageToLogTemplateMapper.cs

```diff
@@ -35,19 +35,19 @@ internal sealed class MessageToLogTemplateMapper : IMessageToLogTemplateMapper
                     }
                 },
                 {
-                    typeof(StudentCreated), new HandlerLogTemplate
+                    typeof(EventCreated), new HandlerLogTemplate
                     {
                         After = "Created a new student with id: {StudentId}."
```

Comment:
I think there we should have message: "Created a new event with id: {EventId}."

Reply:
Should be fixed now, thank you.